setup.py

```diff
@@ -20,7 +20,7 @@
         ]
     },
     install_requires=[
-        'pylexibank>=1.1.1',
+        'pylexibank>=2.0.0',
         'segments==2.0.2'
```

Comment:
Can we unpin this and have >=2.0.2 for segments?

Reply:
Do we even need a pinned version for `segments`? `pylexibank` already informs the system which version it needs (https://github.com/lexibank/pylexibank/blob/021f37481d37b472715033033773543273ed301f/setup.py#L39)

Reply:
Agree, no pinning please.